src/html/Options.html

```diff
@@ -20,7 +20,7 @@ <h1 data-i18n="settings-title"></h1>
               <h3 data-i18n="option_selection"></h3>
               <form>
                   <div>
-                      <select id="type" name="type_select" onchange="admSelectCheck(this);">

```

Comment:
Unfortunately Firefox extensions are subject to [Content Security Policy (CSP) restrictions](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Content_Security_Policy). When Firefox developed WebExtensions following the Chrome extensions API, this behavior is taken from Chrome extensions. 

Basically extensions cannot run JS code outside JS files. This `admSelectCheck(this);` is JS code but it is in the HTML file, so it is forbidden. The change event must be written in another way (see the JS file).

The original answer works in regular websites, but not in extensions.

Reply:
Noted.